src/handlers/transfers/FxFulfilService.js

```diff
@@ -179,13 +234,22 @@ class FxFulfilService {
     }
 
     // Error scenario - fxTransfer.transferStateEnumeration is in some invalid state
-    const fspiopError = fspiopErrorFactory.invalidFxTransferState({ transferStateEnum, action, type })
-    const apiFSPIOPError = fspiopError.toApiErrorObject(this.Config.ERROR_HANDLING)
+    const fspiopError = fspiopErrorFactory.invalidFxTransferState({
+      transferStateEnum,
+      action,
+      type
+    })
+    const apiFSPIOPError = fspiopError.toApiErrorObject(
```

Comment:
I didn't get, why we do code reformatting here, but don't reformat this code:
[As for me, after such reformatting, readability got worse](https://github.com/mojaloop/central-ledger/pull/1057/files#diff-5e8639df73b763ede415b1a0e5bd2931fde236c844a1b34e4e1dd96328e85361R136)

Reply:
@geka-evk Could be the linter in the pre-commit hook. Will fix that.